docs/profiling.md

```diff
@@ -0,0 +1,108 @@
+# vHive profiling tool
+
+The tool focus on the performance characteristics of vHive framework and identify potential 
+performance bottleneck of colocating VMs on a single host. The motivation of this tool is to 
+improve the parallelism of serverless architecture and commercial benefits for cloud vendors.
```

Comment:
Please rephrase the last part. It sounds like we are employees of cloud providers.